### PR TITLE
Misc project cleanup - props, editorconfig, refactorings, docs

### PR DESCRIPTION
**Pull request by martinothamar**
## Description
Spring-cleaning kind of PR. Don't look at the whole diff, only individual commits, hopefully some of them make sense

* Remove .NET Framework reference assemblies package - dev dep I assume is not needed (the ref was >2 years old)
* Cleanup project files in src/ - consistent MSBuild props such as package description, tags. Added README and License to nupkg, hoisted some stuff into props-file
* Docs - Updated README, added LICENSE file, added CONTRIBUTING.md
* Adopted the most common naming conventions (as far as I can tell) as editorconfig naming rules with warn/err diagnostics (diff wasn't too bad)
  * Some places still weird- `WEIRD_CONST_FIELDS`-naming is apparantly still OK for some reason
* Cleaned up unused using (auto codefix)
* File scoped namespaces everywhere (auto codefix)
* Editorconfig formatting for project files (2 spaces, we were a bit all over the place)

This can lead to some nasty conflicts, but seeing as we've just merged some big stuff I figured now might be a good time

## Related Issue(s)
- N/A

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.App.Api/Controllers/ApplicationSettingsController.cs [hunk 1]
```diff
@@ -4,50 +4,46 @@
...
 {
+    private static readonly JsonSerializerOptions _jsonSerializerOptions =
+        new() { DictionaryKeyPolicy = JsonNamingPolicy.CamelCase };
+
+    private readonly AppSettings _appSettings;
+    private readonly FrontEndSettings _frontEndSettings;
+
     /// <summary>
-    /// Controller that exposes application config
+    /// Controller that exposes a subset of app setings
```
Comment (bjorntore) [inline src/Altinn.App.Api/Controllers/ApplicationSettingsController.cs:22]: Nitpick: typo in 'setings'

**Comment by HauklandJ** [comment]
Loading the diff halts chrome, so I am using a comment instead. The `versioning `and `telemetry ` sections of contributing have incomplete sentences.

**Review by HauklandJ — APPROVED** [review]
LGTM - we should have a plan for updating the existing 8 non-renovate PRs before we merge. Should not be to much work though.

**Review by bjorntore — APPROVED** [review]

**Comment by martinothamar** [comment]
> we should have a plan for updating the existing 8 non-renovate PRs before we merge

I can do that I think, where there are conflicts I can just "choose theirs" and redo the auto codefix in the end (fix all)

**Pull request merged**